plutus/budget_manager/requirements.txt

```diff
@@ -0,0 +1,13 @@
+click
```

Comment:
Pin all versions. Capture the output of `pip-freeze`

Reply:
done, this will slightly change once google fixes one of the python apis so ill repin after